plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java

```diff
@@ -57,13 +58,47 @@ public LicenseHeaderConfig licenseHeaderFile(Object licenseHeaderFile) {
 		return licenseHeaderFile(licenseHeaderFile, LICENSE_HEADER_DELIMITER);
 	}
 
-	public void importOrder(String... importOrder) {
-		addStep(ImportOrderStep.forJava().createFrom(importOrder));
+	public ImportOrderConfig importOrder(String... importOrder) {
+		return new ImportOrderConfig(importOrder);
 	}
 
-	public void importOrderFile(Object importOrderFile) {
+	public ImportOrderConfig importOrderFile(Object importOrderFile) {
 		Objects.requireNonNull(importOrderFile);
-		addStep(ImportOrderStep.forJava().createFrom(getProject().file(importOrderFile)));
+		return new ImportOrderConfig(getProject().file(importOrderFile));
+	}
+
+	public class ImportOrderConfig {
+		final String[] importOrder;
+		final File importOrderFile;
+
+		boolean wildcardsLast = false;
+
+		ImportOrderConfig(String[] importOrder) {
+			this.importOrder = importOrder;
+			importOrderFile = null;
+			addStep(createStep());
+		}
+
+		ImportOrderConfig(File importOrderFile) {
+			importOrder = null;
+			this.importOrderFile = importOrderFile;
+			addStep(createStep());
+		}
+
+		/** Sorts wildcard imports after non-wildcard imports, instead of before. */
+		public ImportOrderConfig wildcardsLast() {
```

Comment:
Should follow this pattern

https://github.com/diffplug/spotless/blob/c687694377edfe2a9589f2be90ddb7645f6b40e3/plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java#L119-L126